Automattic/jetpack
# Remove Jetpack Starter activation screen

## Proposed changes:

* Removes Jetpack Starter content from activation screen

### Other information:


## Jetpack product discussion

P2: pbNhbs-7hz-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch and get your local docker environment going (make sure to build `js-packages/licensing` if you do) or spin up an ephemeral site and use the beta plugin to checkout this branch
2. Go to https://wordpress.com/checkout/jetpack/jetpack_starter_yearly and purchase starter
3. Go to your site and activate the product
4. Confirm the activation screen looks like this (Starter not mentioned)
![image](https://github.com/Automattic/jetpack/assets/65001528/43d0e72f-7331-48aa-8de4-ccf49831ec5a)
